CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ The library is built using the notebooks contained in the `nbs` folder. If you w
 
 ### Running tests
 
-* If you're working on the local interface, use `nbdev_test --skip_file_re distributed*`.
+* If you're working on the local interface, use `nbdev_test --skip_file_re "distributed*"`.
```

Comment:
This command changed to skip_file_glob, we should update it as well

Reply:
Fixed in [2e57588](https://github.com/Nixtla/mlforecast/pull/38/commits/2e57588cc881d8401d5899ac1564de1d5d085599) :)